src/main/resources/CliMessages.properties

```diff
@@ -0,0 +1,37 @@
+# Help command messages
+.help.usage.description=Show this help message
+
+# Step command messages
+.step.usage.description=Make the CPU state machine advance 'N' number of steps
+steps.parameter=Number of steps to advance, default is 1
+
+# File command messages
+.file.usage.description=Provide a new file to execute
+file.param=The file to execute
+
+# Dinero command messages
+.dinero.usage.description=Write dinero tracefile to a file
+dinero.param=The file to write to
+
+# Run command messages
+.run.usage.description=Execute the program without intervention
+
+# Show command messages
+.show.usage.description=Show various content of the CPU state machine
```

Comment:
Why are you using both `CurrentLocale.java` and properties files to encode localized text?

Reply:
I was trying to stay consistent and add messages which are output directly from the source in the same manner currently in use. The resource bundles are only used for the PicoCli lib right now. However, I was (forgot to... oops), going to open an issue for converting the entire CurrentLocale to use a resource bundle like the PicoCli lib uses. That way it will be consistent. Thoughts?

Reply:
Makes sense to me! I would prefer to have a single way to store localized strings, and I am perfectly ok to have an intermediate state where the messages are split and an issue to converge to a single mechanism, be it the Java file or the resource bundles. Of the two, of course I prefer the resource bundle, it's more elegant :) thank you